### DISCUSSION TITLE
Workshop fails to start after automatic application update/restart

### DISCUSSION THREAD

**Discussion by 1800joe** [noop-inc/documentation#120] [Bugs]
<img width="330" alt="Screenshot 2024-02-23 at 10 54 21 AM" src="https://github.com/noop-inc/documentation/assets/3111670/ae30a23b-8b4c-45b6-9d8f-f92d773d3dd7">

**Comment by derekwolpert** [comment]
@1800joe I'm unable to replicate this issue when utilizing the auto-updater functionality. Let me know if you have any further context regarding what was occurring on your machine when you encountered this issue.

**Comment by 1800joe** [comment]
Appears fixed in v0.8.2

**Comment by dupuyarc** [comment]
I also experienced this problem (as did @milin82 last night). I uninstalled, used force quit on the app, removed all Noop configuration files from my home directory and ~/Library and everything, re-installed, and the problem didn't clear, it just sat with the spinning circle on the Workshop.

A system restart _did_ resolve the problem. This would lead me to suspect that the problem lies with an already running VM or perhaps some system resource it needs to acquire that was still in use from the old Noop Desktop even after the quit and force quit.

I have attached the `~/Library/Noop/main.log` – I was trying to start the Workshop around noon, and then restarted the system at 12:45 PM. I hope this may help you diagnose the underlying problem that the restart cleared.

[Noop-main.log](https://github.com/noop-inc/documentation/files/14438415/Noop-main.log)

FWIW, I was experiencing the same problem with both 0.8.2 and 0.9.0, and a 0.9.0 installation that was hanging before the system restart was able to start successfully after the system restart.
